src/utils/_list.scss

```diff
@@ -0,0 +1,31 @@
+// --- SUI List --- //
+// @include sui-list;
+// @include sui-list('inline');
+// @include sui-list('grow');
+
+@mixin sui-list-gutter {
+  &:not(:last-child) {
+    margin-right: $m-list-gutter;
+  }
+}
+
+@mixin sui-list($type: 'block'){
+  @include reset-list;
+  display: flex;
+
+  @if $type == 'block' {
+    flex-direction: column;
+  }
+
+  li {
```

Comment:
I really don't like the idea of hace exception. I am prefer to be consistent and style always based in class and never based in tags.